src/components/battle/Detail/BattleMusic/Moving.tsx

```diff
@@ -0,0 +1,45 @@
+import { useRouter } from 'next/router';
+
+import { BattleDetail } from '../../types';
+import BattleMusic from '../BattleMusic';
+
+interface Prop {
+  music: BattleDetail;
+  moving: 'left' | 'right';
+  onClick: (moving: 'left' | 'right', musicId: number | undefined) => void;
+}
+
+function Moving({ music, moving, onClick }: Prop) {
```

Comment:
이거 폴더명에서는 생략해도 컴포넌트명 자체는 풀네임으로 지어주는 게 좋을 것 같아! 가지고 왔을 때 이름 안 바꿔주면 헷갈리니까~

```suggestion
function MovingBattleMusic({ music, moving, onClick }: Prop) {
```